nodestream/pipeline/transformers/transformer.py

```diff
@@ -147,19 +147,19 @@ class SwitchTransformer(Transformer):
     )
 
     @classmethod
-    def from_file_data(cls, switch_on, cases, default, normalization=None):
+    def from_file_data(cls, switch_on, cases, default=None, normalization=None):
         transformers = {
             field_value: ClassLoader().load_class(**transformer)
             for field_value, transformer in cases.items()
         }
-        default = ClassLoader().load_class(**default) if default else PassTransformer
+        default = ClassLoader().load_class(**default) if default else PassTransformer()
         return cls(switch_on, transformers, default, normalization)
 
     def __init__(
         self,
         switch_on: StaticValueOrValueProvider,
         transformers: Dict[str, Transformer],
-        default: Transformer = PassTransformer,
+        default: Transformer = PassTransformer(),
```

Comment:
Default this to None, and create an instance of `PassTransformer` so there isn't a singleton instance.